frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -0,0 +1,88 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import * as loginView from '@console/internal-integration-tests/views/login.view';
+import {
+  withResource,
+  removeLeakedResources,
+  removeLeakableResource,
+} from '@console/shared/src/test-utils/utils';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { createProject } from './utils/utils';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+import { VirtualMachine } from './models/virtualMachine';
+import { VM_ACTION, VM_STATUS } from './utils/consts';
+import { getVMManifest } from './utils/mocks';
+
+const testNonAdminNamespace = `${testName}-non-admin`;
+const isRestrictedAccess = $('.cos-status-box__title');
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 100000;
+const {
+  BRIDGE_HTPASSWD_IDP = 'test',
+  BRIDGE_HTPASSWD_USERNAME = 'test',
+  BRIDGE_HTPASSWD_PASSWORD = 'test',
+} = process.env;
+const configName = ProvisionConfigName.URL;
+const leakedResources = new Set<string>();
+const provisionConfigs = getProvisionConfigs(testName);
+
+const provisionConfig = provisionConfigs.get(configName);
```

Comment:
shouldn't we try to also use nads and storage?

Reply:
does it make sense to initialize config here? Can we couple it with the test?

Reply:
Curretly, this PR automates what we have test plans designed. I agree we should incorporate creation of NAD and DataVolume. I'm okay with addressing that in another PR

Reply:
ok